packages/components/tooltip/stories/Tooltip.stories.tsx

```diff
@@ -212,3 +213,28 @@ export const WithDelays = () => {
     </>
   );
 };
+
+export const WithReactElement = () => {
+  return (
+    <>
+      <SectionHeading as="h3" marginBottom="spacingS">
+        With React Elements as children
+      </SectionHeading>
+
+      <Flex marginBottom="spacingS">
+        <Tooltip
+          isVisible
+          hideDelay={60000}
+          label="Render a paragrah with a React Element"
+          content={
+            <Flex>
+              <p className={css({ marginBottom: 0 })}>I am a React Element</p>
```

Comment:
I think it would be better if we change the example a bit to feature like a headline with a paragraph, to already set a good example when it can be used.
```suggestion
              <div><SectionHeading>Example Heading</SectionHeading><p>I am a React Element</p></div>
```